modules/core/shared/src/main/scala/eu/timepit/refined/types/digests.scala

```diff
@@ -0,0 +1,50 @@
+package eu.timepit.refined.types
+
+import eu.timepit.refined.W
+import eu.timepit.refined.api.{Refined, RefinedTypeOps}
+import eu.timepit.refined.boolean.And
+import eu.timepit.refined.collection.Size
+import eu.timepit.refined.generic.Equal
+
+/** Module for type representing message digests. */
+object digests {
+  import string.HexStringSpec
```

Comment:
The build error on 2.10 can be fixed by moving this import to the top-level imports.

Reply:
I hadn't had time to look at this, thx :)
How did you figure that out? :D

Reply:
I moved and commented stuff out until it compiled. One of my special powers! ;-)